src/types.js

```diff
@@ -0,0 +1,9 @@
+import PropTypes from 'prop-types';
+
+export const CardShape = PropTypes.shape({
+  title: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
```

Comment:
description field is not supposed to be required

Reply:
Fixed!